### PR TITLE
Fix prefix for version validation

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@jiacheliu3 our version string is usually something like hadoop-2.8. this fixes that.

File: integration/tools/validation/src/main/java/alluxio/cli/hdfs/HdfsVersionValidationTask.java [hunk 1]
```diff
@@ -90,6 +92,11 @@ public ValidationTaskResult validateImpl(Map<String, String> optionMap)
     }
 
     String version = mConf.get(PropertyKey.UNDERFS_VERSION);
+    for (String prefix : new String[] {CDH_PREFIX, HADOOP_PREFIX}) {
+      if (version.startsWith(prefix)) {
+        version = version.substring(prefix.length());
```
Comment (jiacheliu3) [inline integration/tools/validation/src/main/java/alluxio/cli/hdfs/HdfsVersionValidationTask.java:97]: nit: maybe `break` if you find a match?

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks for the improvement!

File: integration/tools/validation/src/main/java/alluxio/cli/hdfs/HdfsVersionValidationTask.java (hunk 1 above)
Comment (yuzhu) [reply integration/tools/validation/src/main/java/alluxio/cli/hdfs/HdfsVersionValidationTask.java:97]: done

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**